superset/tasks/schedules.py

```diff
@@ -602,10 +604,16 @@ def deliver_alert(alert: Alert, recipients: Optional[str] = None) -> None:
         textwrap.dedent(
             """\
             <h2>Alert: %(label)s</h2>
-            <img src="cid:screenshot" alt="%(label)s" />
+            <p><b>SQL Statement: </b>%(sql)s</p>
+            <p>Click <a href="%(image_url)s">here</a> or the image below 
```

Comment:
you can probably do even better here: 
https://stackoverflow.com/questions/17245895/how-to-format-text-in-email-when-using-smtp

e.g. 
* highlight sql section & use separate font
* show the result returned
* use red color
* have a link to the alert history

Reply:
also template may live separately, e.g. https://github.com/apache/incubator-superset/blob/master/superset/templates/email/role_extended.txt

Reply:
I'll leave `show the result returned` as a TODO